apiary.apib

```diff
@@ -3808,7 +3808,8 @@ GRANT USAGE ON WAREHOUSE "KEBOOLA" TO ROLE "KEBOOLA_STORAGE" WITH GRANT OPTION;
 
 CREATE USER "KEBOOLA_STORAGE"
     PASSWORD = "YOUR_PASSWORD"
-    DEFAULT_ROLE = "KEBOOLA_STORAGE";
+    DEFAULT_ROLE = "KEBOOLA_STORAGE"
+    TYPE = LEGACY_SERVICE;
```

Comment:
Tady to nemáš oquotované, ale v kódu předtím to quotované bylo

Reply:
Tak vyzkoušej jestli to funguje (já jsem to tady ani tam nezkoušel)

Reply:
![image](https://github.com/user-attachments/assets/0b68eca4-7118-4fe7-bff0-2bc4c427b881)
![image](https://github.com/user-attachments/assets/fdbae6d0-cec5-4cd9-a5a8-897510644d87)